scipion/__main__.py

```diff
@@ -373,7 +373,7 @@ def main():
 
     elif mode in MODE_DEMO:
         runScript(join(SCIPION_SCRIPTS, KICKOFF)
-                  + ' '.join(sys.argv[2:] if len(sys.argv) > 2 else ''))
+                  + ' {}'.format(sys.argv[2:] if len(sys.argv) > 2 else ''))
```

Comment:
Any good reason to change this? We all are used to use join for this ....is this better?, recommended?

Reply:
With the .join approach, it did not detect the empty space character between both elements, and with .format it worked fine